x-pack/plugins/apm/scripts/shared/create_or_update_index.ts

```diff
@@ -1,66 +0,0 @@
-/*
```

Comment:
Unused afaict. 
@dgieselaar Was this used anywhere? You added it but I can no longer find any usages.

Reply:
no, the scripts that used it are gone, so it's fine to delete